pkg/solace/message/message.go

```diff
@@ -114,6 +114,38 @@ type Message interface {
 	//  2 | COS_3
 	GetClassOfService() (cos int)
 
+	// GetCreationTraceContext will return the trace context metadata used for distributed message tracing message
+	// creation context information across service boundaries.
+	// It allows correlating the producer with the consumers of a message, regardless of intermediary
+	// instrumentation. It must not be altered by intermediaries.
+	// If the content is not accessible, an empty slice will be returned and the ok flag will be false.
+	GetCreationTraceContext() (traceID [16]byte, spanID [8]byte, sampled bool, traceState string, ok bool)
```

Comment:
all otel methods need to be removed from a public interface

Reply:
these will be removed